bridges/snowbridge/runtime/test-common/src/lib.rs

```diff
@@ -561,3 +561,46 @@ pub fn ethereum_to_polkadot_message_extrinsics_work<Runtime>(
 			assert_ok!(sync_committee_outcome);
 		});
 }
+
+pub fn fetch_delivery_cost<Runtime, XcmConfig>(collator_session_key: CollatorSessionKeys<Runtime>)
+where
+	Runtime: frame_system::Config
+		+ pallet_balances::Config
+		+ pallet_session::Config
+		+ pallet_xcm::Config
+		+ parachain_info::Config
+		+ pallet_collator_selection::Config
+		+ cumulus_pallet_parachain_system::Config
+		+ snowbridge_pallet_outbound_queue::Config
+		+ snowbridge_pallet_inbound_queue::Config
+		+ pallet_timestamp::Config,
+	XcmConfig: xcm_executor::Config,
+	ValidatorIdOf<Runtime>: From<AccountIdOf<Runtime>>,
+	<Runtime as frame_system::Config>::AccountId: From<sp_runtime::AccountId32> + AsRef<[u8]>,
+{
+	ExtBuilder::<Runtime>::default()
+		.with_collators(collator_session_key.collators())
+		.with_session_keys(collator_session_key.session_keys())
+		.with_para_id(1013.into())
+		.with_tracing()
+		.build()
+		.execute_with(|| {
+			let _ = <pallet_xcm::Pallet<Runtime>>::force_xcm_version(
+				RuntimeHelper::<Runtime>::root_origin(),
+				Box::new(Location { parents: 1, interior: [Parachain(1000)].into() }),
+				XCM_VERSION,
+			);
+			let inbound_delivery_cost =
+				<snowbridge_pallet_inbound_queue::Pallet<Runtime>>::calculate_delivery_cost(
+					<Runtime as snowbridge_pallet_inbound_queue::Config>::MaxMessageSize::get(),
+				);
+			let inbound_send_cost =
+				<Runtime as snowbridge_pallet_inbound_queue::Config>::MaxSendCost::get();
```

Comment:
We should probably assert that the configured cost is greater than or equal to the xcm cost using the same method you used here:

https://github.com/Snowfork/polkadot-sdk/pull/142/commits/8ad87ba8f861f8020752d52063b017423c80553f#diff-832917c2fd6302a9f175210f7d9941ea82a1a7c5a35aa35e662380074d405203L370-L377

Probably not important for this test. But may be very important for the fellowship runtime.

Reply:
https://github.com/Snowfork/polkadot-sdk/pull/142/commits/4012a59586b251ecfff74f4f83bef0b59b4f3977